test/exit-format-sol.test.ts

```diff
@@ -51,4 +57,42 @@ describe("ExitFormat (solidity)", function () {
       "0x00000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000060000000000000000000000000000000000000000000000000000000000000008000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000002000000000000000000000000096f7123e3a80c9813ef50213aded0e4511cb820f0000000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000800000000000000000000000000000000000000000000000000000000000000000"
     );
   });
+
+  it("Can execute a single asset exit and a whole exit", async function () {
+    const amount = "0x01";
+
+    const alice = new Wallet(
+      "0x68d3e3134e2b3488ad249233f8fa77ea040bbb6434ea28e4acde7db082665c4c"
+    );
+
+    const singleAssetExit: SingleAssetExit = {
+      asset: "0x0000000000000000000000000000000000000000",
+      metadata: "0x",
+      allocations: [
+        {
+          destination: "0x000000000000000000000000" + alice.address.slice(2), // padded alice
+          amount,
+          allocationType: AllocationType.simple,
+          metadata: "0x",
+        },
+      ],
+    };
+
+    await testConsumer.signer.sendTransaction({
+      to: testConsumer.address,
+      value: BigNumber.from(amount).mul(2),
```

Comment:
I think I get the motivation here (start from a given amount, add twice that much, withdraw the original twice, then check that balances are as they should be) but it took a few minutes for me to get there. Maybe that's not bad? Or maybe the sequence of balances could be computed up front, and their 'togetherness' would improve readability:

``` ts
const withdrawlSize = BigNumber.from("0x01");
const totalDeposit = withdrawSize.mul(2);
```
and use these constants in the test?